src/icons/icon-success.tsx

```diff
@@ -28,26 +28,35 @@ const IconSuccessO2: React.FC<Props> = ({size = 64, color, skipAnimation = false
         <svg role="presentation" width={size} height={size} viewBox="0 0 64 64">
             <g
                 className={classnames({[classes.iconContainer]: !color})}
-                strokeWidth="2"
                 strokeLinecap="round"
                 strokeLinejoin="round"
             >
```

Comment:
Check animation have a little buggy translation

https://user-images.githubusercontent.com/6722153/150359271-1702ed89-c40f-4283-aaea-96b3aafc365b.mov

Reply:
Do you mean that little 'hop' that the check makes?

Reply:
yep!

Reply:
Check if now it suits up!

Reply:
nice! currazo!